platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/reconnect/lag/BenchmarkSlowLearningSynchronizer.java

```diff
@@ -52,8 +53,9 @@ public BenchmarkSlowLearningSynchronizer(
             final long delayNetworkMicroseconds,
             final double delayNetworkFuzzRangePercent,
             final Runnable breakConnection,
-            final ReconnectConfig reconnectConfig) {
-        super(getStaticThreadManager(), in, out, root, breakConnection, reconnectConfig);
+            final ReconnectConfig reconnectConfig,
+            final Metrics metrics) {
```

Comment:
missing nullity annotation for new param

Reply:
added